.rubocop_todo.yml

```diff
@@ -440,6 +440,8 @@ RSpec/MessageSpies:
 # Offense count: 131
 RSpec/MultipleExpectations:
   Max: 14
+RSpec/NestedGroups:
+  Max: 4 # previously max 3
```

Comment:
comment explaining "why" ... doesn't explain "why".   Maybe "tests readable with this" or possibly better:  exclude the particular failing file, and comment "why" for that specific file.